# Translation of Odoo Server.
# This file contains the translation of the following modules:
# 	* vitt_CAE_recover
#
msgid ""
msgstr ""
"Project-Id-Version: Odoo Server 10.0+e\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2019-03-15 14:58+0000\n"
"PO-Revision-Date: 2019-03-15 12:03-0300\n"
"Last-Translator: <>\n"
"Language-Team: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: \n"
"Language: es\n"
"X-Generator: Poedit 2.2.1\n"

#. module: vitt_CAE_recover
#: code:addons/vitt_CAE_recover/models/account_journal_document_type.py:45
#, python-format
msgid "AFIP WS %s not implemented"
msgstr "WS AFIP %s aún no ha sido implementado"

#. module: vitt_CAE_recover
#: model:ir.model,name:vitt_CAE_recover.model_afip_ws_consult_wizard
msgid "AFIP WS Consult Wizard"
msgstr "Asistente de consulta de WS AFIP"

#. module: vitt_CAE_recover
#: model:ir.model.fields,field_description:vitt_CAE_recover.field_account_invoice_caerecover_create_uid
msgid "Created by"
msgstr "Creado por"

#. module: vitt_CAE_recover
#: model:ir.model.fields,field_description:vitt_CAE_recover.field_account_invoice_caerecover_create_date
msgid "Created on"
msgstr "Creado el"

#. module: vitt_CAE_recover
#: model:ir.model.fields,field_description:vitt_CAE_recover.field_account_invoice_caerecover_display_name
msgid "Display Name"
msgstr "Nombre"

#. module: vitt_CAE_recover
#: model:ir.model.fields,field_description:vitt_CAE_recover.field_account_invoice_caerecover_doctype_id
msgid "Document Type"
msgstr "Tipo de Documento"

#. module: vitt_CAE_recover
#: model:ir.ui.view,arch_db:vitt_CAE_recover.account_invoice_caerecover_form
msgid "Excel Report file"
msgstr "Archivo Informe en Excel"

#. module: vitt_CAE_recover
#: model:ir.model.fields,field_description:vitt_CAE_recover.field_account_invoice_caerecover_id
msgid "ID"
msgstr "ID"

#. module: vitt_CAE_recover
#: code:addons/vitt_CAE_recover/models/account_journal_document_type.py:47
#, python-format
msgid "Invoice number %s\n"
msgstr "Número Factura %s\n"

#. module: vitt_CAE_recover
#: model:ir.model.fields,field_description:vitt_CAE_recover.field_account_invoice_caerecover_journal_id
msgid "Journal"
msgstr "Diario"

#. module: vitt_CAE_recover
#: model:ir.model,name:vitt_CAE_recover.model_account_journal_document_type
msgid "Journal Document Types Mapping"
msgstr "Mapeo de Tipos de Documentos del Diario"

#. module: vitt_CAE_recover
#: model:ir.model.fields,field_description:vitt_CAE_recover.field_account_invoice_caerecover___last_update
msgid "Last Modified on"
msgstr "Ult. Modificación en"

#. module: vitt_CAE_recover
#: model:ir.model.fields,field_description:vitt_CAE_recover.field_account_invoice_caerecover_write_uid
msgid "Last Updated by"
msgstr "Ult. Actualización por"

#. module: vitt_CAE_recover
#: model:ir.model.fields,field_description:vitt_CAE_recover.field_account_invoice_caerecover_write_date
msgid "Last Updated on"
msgstr "Ult. Actualización el"

#. module: vitt_CAE_recover
#: code:addons/vitt_CAE_recover/wizard/wizard.py:63
#, python-format
msgid "Los datos no corresponden con ninguna dentro del sistema"
msgstr "Los datos no corresponden con ninguna dentro del sistema"

#. module: vitt_CAE_recover
#: code:addons/vitt_CAE_recover/models/account_journal_document_type.py:23
#, python-format
msgid "No AFIP WS selected on point of sale %s"
msgstr "No hay un WS AFIP seleccionado para el pto. venta %s"

#. module: vitt_CAE_recover
#: code:addons/vitt_CAE_recover/models/afip_ws_consult_wizard.py:30
#, python-format
msgid "No Journal Document Class as active_id on context"
msgstr "No hay una Clase de Documento Diario como active_id en contexto"

#. module: vitt_CAE_recover
#: code:addons/vitt_CAE_recover/models/afip_ws_consult_wizard.py:41
#: code:addons/vitt_CAE_recover/wizard/wizard.py:24
#, python-format
msgid "No existen datos de esa factura en la Afip"
msgstr "No existen datos que correspondan a esa factura en la AFIP"

#. module: vitt_CAE_recover
#: code:addons/vitt_CAE_recover/models/afip_ws_consult_wizard.py:44
#: code:addons/vitt_CAE_recover/wizard/wizard.py:27
#, python-format
msgid "No se pudo retraer CAE"
msgstr "No se pudo recuperar CAE"

#. module: vitt_CAE_recover
#: model:ir.model.fields,field_description:vitt_CAE_recover.field_account_invoice_caerecover_number
msgid "Number (EJ: 0021-00010695)"
msgstr "Número (Ej: 695)"

#. module: vitt_CAE_recover
#: model:ir.actions.act_window,name:vitt_CAE_recover.group_cae_recover_act
msgid "Recover CAE"
msgstr "Recuperar Cód. CAE"

#. module: vitt_CAE_recover
#: model:ir.ui.view,arch_db:vitt_CAE_recover.account_invoice_caerecover_form
msgid "Recover CAEs"
msgstr "Recuperar Cód. CAEs"

#. module: vitt_CAE_recover
#: model:ir.ui.view,arch_db:vitt_CAE_recover.new_invoice_form2
msgid "Recuperar CAE"
msgstr "Recuperar Cód. CAE"

#. module: vitt_CAE_recover
#: model:ir.model,name:vitt_CAE_recover.model_account_invoice_caerecover
msgid "account.invoice.caerecover"
msgstr "account.invoice.caerecover"

#. module: vitt_CAE_recover
#: code:addons/vitt_CAE_recover/models/afip_ws_consult_wizard.py:71
#, python-format
msgid "la factura no corresponde con los datos obtenidos"
msgstr "Los datos de la factura no corresponden con los datos obtenidos"

#. module: vitt_CAE_recover
#: code:addons/vitt_CAE_recover/models/afip_ws_consult_wizard.py:26
#, python-format
msgid "solo sin CAE"
msgstr "Solo para facturas sin CAE"

#. module: vitt_CAE_recover
#: code:addons/vitt_CAE_recover/models/afip_ws_consult_wizard.py:48
#: code:addons/vitt_CAE_recover/wizard/wizard.py:31
#, python-format
msgid "you have an invoice with this CAE: %s"
msgstr "Ud. ya tiene una factura con este código CAE: %s"
